tests/integration_tests/dashboards/filter_state/api_tests.py

```diff
@@ -0,0 +1,157 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+from flask.testing import FlaskClient
+from sqlalchemy.orm import Session
+
+from superset import app
+from superset.dashboards.commands.exceptions import DashboardAccessDeniedError
+from superset.extensions import cache_manager
+from superset.key_value.utils import cache_key
+from superset.models.dashboard import Dashboard
+from tests.integration_tests.fixtures.world_bank_dashboard import (
+    load_world_bank_dashboard_with_slices,
+)
+from tests.integration_tests.test_app import app
```

Comment:
`app` is reimported here.
mypy hook should have given error in ci too as my local does.
not sure how it was skipped.

Reply:
Thanks for pointing that out @mayurnewase. I submitted a fix in https://github.com/apache/superset/pull/17887. @villebro Do you have any idea how `pre-commit` and CI missed that?